src/main/java/ch/njol/skript/config/Node.java

```diff
@@ -97,7 +104,31 @@ public String getKey() {
 	public final Config getConfig() {
 		return config;
 	}
-	
+
+	/**
+	 * Returns the node that comes before this in the structure.
+	 * Note that a parent is <b>not</b> considered a previous node
+	 * and thus once you have reached {@code null} you should get
+	 * the parent via {@link Node#getParent()} if you wish to continue
+	 * traversing the structure.
+	 */
+	@Nullable
+	public Node getPrevious() {
+		return previous;
+	}
+
+	/**
+	 * @return the node that comes after this in the structure
+	 */
+	@Nullable
+	public Node getNext() {
+		return next;
+	}
+
+	public int getLineNum() {
```

Comment:
You should just make this `getLineNumber` or `getLine`.

Edit: Oh noice I can make reviews now.